app/controllers/descriptives_controller.rb

```diff
@@ -17,8 +39,13 @@ def show
 
   private
 
-  def load_cocina
-    @cocina = Dor::Services::Client.object(params[:item_id]).find
+  def load_and_authorize_cocina
+    @cocina = dor_client.find
+    authorize! :manage_item, @cocina
+  end
+
+  def dor_client
+    Dor::Services::Client.object(params[:item_id])
   end
```

Comment:
if these methods are going to be repeated in other controllers (like here: https://github.com/sul-dlss/argo/blob/main/app/controllers/structures_controller.rb#L32-L39), thoughts on moving to the application controller to avoid duplication?

Reply:
item_id isn't always the parameter to use. I find that having this in the class makes it easier to reason about.  This class isn't very long and there isn't much chance of getting this call wrong.

Reply:
I've addressed this duplication in a more systematic way via https://github.com/sul-dlss/argo/pull/3406